tutorcodejail/plugin.py

```diff
@@ -0,0 +1,48 @@
+from glob import glob
+import os
+import pkg_resources
+
+from .__about__ import __version__
+
+templates = pkg_resources.resource_filename(
+    "tutorcodejail", "templates"
+)
+
+config = {
+    "add": {
+        "SECRET_KEY": "{{ 24|random_string }}",
+    },
+    "defaults": {
+        "VERSION": __version__,
+        "HOST": "codejailservice",
+        "DOCKER_IMAGE": f"docker.io/ednxops/codejailservice:{__version__}",
+    },
+    "set":{
+        "SANDBOX_PYTHON_VERSION": "3.5.10",
+    }
+}
+
+hooks = {
+    "build-image": {
+        "codejail": "{{ CODEJAIL_DOCKER_IMAGE }}",
+        "codejail_apparmor": "docker.io/ednxops/codejail_apparmor:latest"
```

Comment:
I suggest to replace ":latest" by the actual plugin version. This guarantees that the Docker image is always pinned. This is why I always add a "VERSION" setting. See for instance: https://github.com/overhangio/tutor-mfe/blob/932f2db9fdf4c9370abeec5e89a558d9f590f133/tutormfe/plugin.py#L13